drivers: display: Add SSD1325 Support (#102128)
zephyrproject-rtos/zephyr: doc/releases/migration-guide-4.4.rst

     GPT now uses explicit devicetree properties rather than hardcoded values, allowing
     per-instance customization.

Display
=======

> * SSD1327 driver and relevant variables have been renamed SSD1327_5 to handle SSD1325.

Author: Well it does list it, just not after your proposed change (SSD1327 -> SSD1327_5 is the only symbol name change since the contrast and size has been separated).

> * SSD1327 driver and relevant variables have been renamed SSD1327_5 to handle SSD1325.

Author: done (made it detailed)

> * SSD1327 binding has been renamed :dtcompatible:`solomon,ssd1327` from ``solomon,ssd1327fb``

Author: done

* For ILI9341 controller, display mirroring configuration has been updated to conform with
  the described behavior of the sample ``samples/drivers/display``. (:github:`99267`).

> * SSD1327 Kconfig symbols and files have been renamed to include SSD1325 as well:

Reviewer: ```suggestion
* SSD1327 Kconfig symbols have been renamed to include SSD1325 as well:
```

   * Driver Kconfig SSD1327 becomes SSD1327_5.

>    * All names in code go from SSD1327 to SSD1327_5 unless specific to one of the controllers.

Reviewer: ```suggestion
   * ``CONFIG_SSD1327`` -> :kconfig:option:`CONFIG_SSD1327_5`.
```

Author: done

   * Added SSD1325_DEFAULT_CONTRAST and SSD1325_CONV_BUFFER_LINES symbols for SSD1325.

* ``solomon,ssd1327fb`` devicetree compatible has been renamed :dtcompatible:`solomon,ssd1327`

>   to harmonize with other drivers and eliminate the zephyr-irrelevant ``fb`` suffix.

Reviewer: ```suggestion
* ``solomon,ssd1327fb`` devicetree compatible has been renamed to :dtcompatible:`solomon,ssd1327`
  to harmonize with other display controllers and eliminate the zephyr-irrelevant ``fb`` suffix.
```
